spec/components/collections/edit/pane_component_spec.rb

```diff
@@ -26,12 +30,32 @@
         component.with_deposit_button { '<button>Deposit</button>'.html_safe }
       end) { '<div>Test Pane Content</div>'.html_safe }
       tab_pane = page.find('div.tab-pane')
-      expect(tab_pane).to have_no_button('Next')
-      expect(tab_pane).to have_link('Cancel')
```

Comment:
this is now tested separately below, since they are independent of the deposit button logic now